HACKING.md

```diff
@@ -106,6 +106,47 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
```

Comment:
1. Set a `URL` to your server in `/etc/libreport/plugins/ureport.conf`
```
URL = http://localhost:8080/faf
```

```markdown
1. Set a `URL` to your server in `/etc/libreport/plugins/ureport.conf`
\```
URL = http://localhost:8080/faf
\```
```